backend/src/main/java/ch/puzzle/okr/service/MeasureService.java

```diff
@@ -23,6 +23,19 @@ public Measure saveMeasure(Measure measure) {
         if (measure.getId() != null) {
             throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Measure has already an Id");
         }
+        this.checkMeasure(measure);
+        return measureRepository.save(measure);
+    }
+
+    public Measure updateMeasure(Long id, Measure measure) {
+        this.measureRepository.findById(id)
+                .orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND,
+                        "Measure with this id wasn't found"));
```

Comment:
id in Response zurückgeben. Wie auf Zeile 59

Reply:
String.format("Measure with id %d not found", id) verwendet um 
 nun auch id in Exception zurückzugeben